Zilon.Core/Zilon.Core.Tests/Persons/EquipmentModuleTests.cs

```diff
@@ -562,6 +562,64 @@ public void SetEquipment_PistolAndShield2_NoException()
             }
         }
 
+        /// <summary>
+        /// Тест проверяет, что можно при экипировке двуручного меча можно его заменить в любую руку.
+        /// </summary>
+        [Test]
+        public void SetEquipment_ReplacedGreatSword_NoException()
+        {
+            // ARRANGE
+            var greatSwordScheme = new TestPropScheme
+            {
+                Tags = new[] { PropTags.Equipment.Weapon, PropTags.Equipment.WeaponClass.Sword },
+                Equip = new TestTwoHandPropEquipSubScheme
+                {
+                    SlotTypes = new[]
+                    {
+                        EquipmentSlotTypes.Hand
+                    }
+                }
+            };
+
+            var slotSchemes = new[]
+            {
+                new PersonSlotSubScheme
+                {
+                    Types = EquipmentSlotTypes.Hand
+                },
+                new PersonSlotSubScheme
+                {
+                    Types = EquipmentSlotTypes.Hand
+                }
+            };
+
+            var tacticalActScheme = new TestTacticalActScheme
+            {
+                Stats = new TestTacticalActStatsSubScheme
+                {
+                    Range = new Range<int>(1, 1)
+                }
+            };
+
+            var greatSwordEquipment1 = new Equipment(greatSwordScheme, new[] { tacticalActScheme });
+            var greatSwordEquipment2 = new Equipment(greatSwordScheme, new[] { tacticalActScheme });
+
+            const int swordSlot1 = 0;
+            const int swordSlot2 = 1;
+
+            var carrier = new EquipmentModule(slotSchemes);
+
+            // ACT
+            Action act = () =>
+            {
+                carrier[swordSlot1] = greatSwordEquipment1;
+                carrier[swordSlot2] = greatSwordEquipment2;
+            };
+
+            // ASSERT
+            act.Should().NotThrow<Exception>();
```

Comment:
It's better to check the second instance of two-handed weapon in the slot 2 and null in the slot 1 instead just exception throwing.